tests/cli/commands/test_config_command.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from click.testing import CliRunner
+
+from kibble.cli.commands.config_command import config_group
+
+
+class TestConfigCommand:
+    def test_show(self):
+        runner = CliRunner()
+        result = runner.invoke(config_group, ["show"])
+
+        assert result.exit_code == 0
+        assert result.output.strip() == "To be implemented!"
```

Comment:
Similary to the command - those are here mostly as reference where and how to write tests